api/src/test/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirR3ResourceProviderTest.java

```diff
@@ -212,4 +218,14 @@ public void searchConditions_shouldReturnConditionReturnedByServiceWhenPatientIs
 		assertThat(resultList.size(), greaterThanOrEqualTo(1));
 		assertThat(resultList.iterator().next().fhirType(), equalTo(FhirConstants.CONDITION));
 	}
+	
+	@Delete
+	public OperationOutcome deleteObservationResource(@IdParam @NotNull IdType id) {
+		org.hl7.fhir.r4.model.Condition condition = conditionService.delete(id.getIdPart());
```

Comment:
@sherrif10  we have to mock here not directly call the service method

Reply:
Actually @Akayeshmantha  i tried adding the mock but in resolving stubbing tests